sandbox/lua/modules/graphite.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Module contains utilities for Graphite
+--]]
+-- Imports
+require "string"
+require "math"
+require "table"
+
+local M = {}
+
```

Comment:
I was playing with some code to address 1496 but the original hack of not using environments will have to stand for now.